lib/python/milc.py

```diff
@@ -519,7 +529,10 @@ def merge_args_into_config(self):
                         self.config[section][argument] = arg_value
                 else:
                     if argument not in self.config[section]:
-                        self.config[section][argument] = getattr(self.args, argument)
+                        # Check if the argument exist for this section
+                        arg = getattr(self.args, argument)
+                        if arg:
```

Comment:
I have the same concern here. Is there a reason to filter out all false values and not just None?

Reply:
I don't think so, therefore in favour of `if arg is not None:`.